src/shared/components/TextArea/TextArea.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { forwardRef, useState } from 'react'
-import { getVariant } from '../InputBase'
-import { HelperText, HelperTextCount, HelperTextsWrapper, StyledTextArea, TextAreaWrapper } from './TextArea.style'
+import InputBase, { InputBaseProps } from '../InputBase'
+import { StyledTextArea } from './TextArea.style'
 
 export type TextAreaProps = {
```

Comment:
Some of props specified here are already defined in `InputBaseProps`, let's try not to duplicate those here